stuff/cron/update_ranks.py

```diff
@@ -181,6 +186,64 @@ def update_user_rank_cutoffs(cur: MySQLdb.cursors.BaseCursor,
                      cutoff.percentile, cutoff.classname))
 
 
+def update_school_rank(cur: MySQLdb.cursors.BaseCursor) -> None:
+    '''Updates the school rank'''
+
+    logging.info('Updating school rank...')
+    cur.execute('''
+        SELECT
+            s.school_id,
+            SUM(ROUND(100 / LOG(2, distinct_school_problems.accepted+1), 0))
+            AS score
+        FROM
+            Schools s
+        INNER JOIN
+            (
+                SELECT
+                    su.school_id,
+                    p.accepted,
+                    MIN(su.time)
+                FROM
+                    Submissions su
+                INNER JOIN
+                    Runs r ON r.run_id = su.current_run_id
+                INNER JOIN
+                    Problems p ON p.problem_id = su.problem_id
+                WHERE
+                    r.verdict = "AC"
+                    AND p.visibility >= 1
+                    AND su.school_id IS NOT NULL
+                GROUP BY
+                    su.school_id,
+                    su.problem_id
+            ) AS distinct_school_problems
+        ON
+            distinct_school_problems.school_id = s.school_id
+        GROUP BY
+            s.school_id
+        ORDER BY
+            score DESC;
+    ''')
+
+    rank = 0
+    prev_score = None
+
+    for row in cur:
+        if row['score'] != prev_score:
+            rank += 1
```

Comment:
```suggestion
    for index, row in enumerate(cur):
        if row['score'] != prev_score:
            rank = index + 1
```
y ahora que lo pienso, parece que L148 tiene el mismo bug D: puedes corregirlo también?